modules/router/src/listener.ts

```diff
@@ -166,12 +167,44 @@ export async function setupListeners(
           "Error forwarding resolution",
         );
       }
-      logger.info({ method: "forwardTransferResolution", result: res.getValue() }, "Successfully forwarded resolution");
+      logger.info(
+        { event: EngineEvents.CONDITIONAL_TRANSFER_RESOLVED, result: res.getValue() },
+        "Successfully forwarded resolution",
+      );
+
+      const transferSenderResolutionChannelAddress = res.getValue()?.channelAddress;
+      const transferSenderResolutionTransferId = res.getValue()?.transferId;
+      if (!transferSenderResolutionChannelAddress || !transferSenderResolutionTransferId) {
+        logger.warn(
+          {
+            event: EngineEvents.CONDITIONAL_TRANSFER_RESOLVED,
+            transferSenderResolutionChannelAddress,
+            transferSenderResolutionTransferId,
```

Comment:
should probably just log what the response is here

Reply:
We do two lines above.

Reply:
ah should've expanded the preview, my bad